packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -71,17 +71,15 @@ export function moduleFor(description, TestClass, ...mixins) {
     }
 
     if (name.indexOf('@test ') === 0) {
-      QUnit.test(name.slice(5), assert => context[name](assert));
+      QUnit.test(name.slice(5), assert => context[name](context.assert));
     } else if (name.indexOf('@skip ') === 0) {
-      QUnit.skip(name.slice(5), assert => context[name](assert));
+      QUnit.skip(name.slice(5), assert => context[name](context.assert));
     } else if (name.indexOf(packageTag) === 0) {
-      QUnit.test(name.slice(packageTag.length), assert => context[name](assert));
+      QUnit.test(name.slice(packageTag.length), assert => context[name](context.assert));
```

Comment:
the original code is fine - the assert actually comes as a method argument from the caller (the QUnit runner), not from the global scope. The es6 arrow function syntax probably made that a bit too hidden :)